code/upaya/app/assets/stylesheets/main.scss

```diff
@@ -1,2 +1,2 @@
-@import "us_web_design_standards";
-@import "us_web_design_standards_fonts";
\ No newline at end of file
+/*@import "us_web_design_standards";
+@import "us_web_design_standards_fonts";*/
```

Comment:
Hmm. That goes against the sass-rails recommendation and how I've seen SASS being used in Rails. https://github.com/rails/sass-rails#important-note